.github/workflows/release.yml

```diff
@@ -34,3 +35,39 @@ jobs:
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
           PLUGIN_PORTAL_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
           PLUGIN_PORTAL_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
+
+  release-docs:
+    runs-on: ubuntu-latest
+    if: github.repository == 'ExpediaGroup/graphql-kotlin'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('website/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Build website
+        run: |
+          NEW_VERSION=$(echo "${GITHUB_REF}" | cut -d "/" -f3)
+          echo "New version: ${NEW_VERSION}"
+          cd website
+          npm install
+          npm run version $NEW_VERSION
```

Comment:
this actually creates new folder/files under `website` dir, if you want to auto publish those then I guess we need to explicitly add them to the repo? I believe the next action only pushes to `gh-pages`  branch.

Reply:
This action here does publish to the `gh-pages` branch

https://github.com/ExpediaGroup/graphql-kotlin/pull/852/files#diff-56c30d9224897b08c28075052d4595c5R68

Reply:
yes action pushes to `gh-pages` but we need to push to master - update sidebars (https://github.com/ExpediaGroup/graphql-kotlin/tree/master/website/versioned_sidebars) and create versioned markdown files (https://github.com/ExpediaGroup/graphql-kotlin/tree/master/website/versioned_docs/version-3.6.1)

Reply:
Oh, right. I will update to use the following action: https://github.com/marketplace/actions/github-push?version=v0.6.0